client/state/sites/selectors/get-site-computed-attributes.js

```diff
@@ -4,7 +4,7 @@
 import { withoutHttp } from 'calypso/lib/url';
 import canCurrentUser from 'calypso/state/selectors/can-current-user';
 import getRawSite from 'calypso/state/selectors/get-raw-site';
-import getSiteOptions from 'calypso/state/selectors/get-site-options';
+import getSiteOptions from './get-site-options';
```

Comment:
Nit: if we're keeping these alphabetized, it could be moved below `getSiteOption`.